$NetBSD: patch-CMakeLists.txt,v 1.1 2011/12/27 10:08:50 absd Exp $

Broken version compare semantics means 1.10 is take to be less than 1.6

--- CMakeLists.txt.orig	2011-12-23 08:55:00.000000000 +0000
+++ CMakeLists.txt
@@ -25,7 +25,8 @@ macro_log_feature(SWSCALE_FOUND "swscale
 macro_optional_find_package(Xine)
 macro_log_feature(XINE_XCB_FOUND "Xine XCB" "Xine libraries" "http://www.xine-project.org/" FALSE "" "Xine and phonon-xine must be compiled with XCB support.")
 
-macro_optional_find_package(Taglib 1.6)
+# Broken version compare semantics means 1.10 is take to be less than 1.6
+macro_optional_find_package(Taglib 1.10)
 macro_log_feature(TAGLIB_FOUND "taglib" "Library for reading and editing audio meta data." "http://ktown.kde.org/~wheeler/taglib/" FALSE "1.6" "Required to build Juk and many Strigi Analyzers.")
 
 macro_optional_find_package(OggVorbis)
